CONTRIBUTING.md

```diff
@@ -0,0 +1,35 @@
+## Update manifests
+
+KFP charms use Jinja2 templates in order to store manifests that need to be applied during its deployment. The process for updating them is:
+
+### Spot the differences between versions of a manifest file
+
+1. Install `kustomize` using the official documentation [instructions](https://kubectl.docs.kubernetes.io/installation/kustomize/)
+2. Clone [Kubeflow manifests](https://github.com/kubeflow/manifests) repo locally
+3. `cd` into the repo and checkout to the branch or tag of the target version.
+4. Build the manifests with `kustomize` and save the file:
+
+```bash
+kustomize build apps/pipeline/upstream/env/cert-manager/platform-agnostic-multi-user > kfp-manifests-vX.yaml
```

Comment:
Since this may change in the future, would it make sense to add a link to the pipelines section of the readme of `kubeflow/manifests` repo to show what we take as reference?